src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java

```diff
@@ -195,7 +202,7 @@ private Field schemaToField(Schema sch) {
       valueType = subField.getValueType();
     }
 
-    Field field = new Field(name, valueType, repeated, keyType, description);
+    Field field = new Field(name, valueType, repeated, keyType, sanitizeDescr(description));
```

Comment:
This and the previous usage of `sanitizeDexcr` reference the same description, right? So maybe assigned the sanitized description at the top of the method (and s/`description`/`sanitizedDescription`/)

Reply:
Yes, we can do it, but doing it the current way lets us keep the sanitization logic consistent (i.e. sanitization method is called right before calling the constructor of a message/field/service/method).